### PR TITLE
BBB-184 Introduce actuator end points. Move to yml config

### PR DESCRIPTION
**Pull request by stephen-bealine**

### CODE REVIEW

File: acceptance-tests/src/test/java/uk/gov/service/bluebadge/test/acceptance/pages/site/AbstractSitePage.java [hunk 1]
```diff
@@ -6,6 +6,7 @@
 public abstract class AbstractSitePage extends AbstractPage {
 
   public static final String URL = System.getProperty("webdriver.base.url");
+  public static final String actuator_URL = System.getProperty("webdriver.actuator.url");
```
Comment (mgilgar) [inline acceptance-tests/src/test/java/uk/gov/service/bluebadge/test/acceptance/pages/site/AbstractSitePage.java]: maybe ACTUATOR_URL

File: src/test/java/uk/gov/dft/bluebadge/webapp/la/integration/ActuatorTests.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package uk.gov.dft.bluebadge.webapp.la.integration;
```
Comment (mgilgar) [inline src/test/java/uk/gov/dft/bluebadge/webapp/la/integration/ActuatorTests.java]: Is this a regular unit test? Will it be run when gradle build?

File: src/test/java/uk/gov/dft/bluebadge/webapp/la/integration/IntegrationTestsBase.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
+package uk.gov.dft.bluebadge.webapp.la.integration;
```
Comment (mgilgar) [inline src/test/java/uk/gov/dft/bluebadge/webapp/la/integration/IntegrationTestsBase.java]: Why are we changing to rest assured? What is the advantage?

File: src/test/java/uk/gov/dft/bluebadge/webapp/la/integration/ActuatorTests.java (hunk 2 above)
Comment (stephen-bealine) [reply src/test/java/uk/gov/dft/bluebadge/webapp/la/integration/ActuatorTests.java]: yes

File: acceptance-tests/src/test/java/uk/gov/service/bluebadge/test/acceptance/pages/site/AbstractSitePage.java (hunk 1 above)
Comment (stephen-bealine) [reply acceptance-tests/src/test/java/uk/gov/service/bluebadge/test/acceptance/pages/site/AbstractSitePage.java]: File deleted

**Pull request merged**